### PR TITLE
i18n preset

### PR DESCRIPTION
**Pull request by alexgul4enko**

### CODE REVIEW

**Comment by alexgul4enko** [comment]
also we can add https://github.com/yanivkalfa/i18n-gettext-webpack-plugin to parse translation on FE part

File: presets/i18n.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import { ProvidePlugin } from 'webpack'
+
+export default function(config) {
+  return addPlugins([
+    new ProvidePlugin({
+      gettext: ['ds-frontend/packages/i18n', 'gettext'],
+      pgettext: ['ds-frontend/packages/i18n', 'pgettext'],
+      ngettext: ['ds-frontend/packages/i18n', 'ngettext'],
+      npgettext: ['ds-frontend/packages/i18n', 'npgettext'],
+      gettextNoop: ['ds-frontend/packages/i18n', 'gettextNoop'],
```
Comment (legendar) [inline presets/i18n.js:11]: what the reason for this method? 
`gettext` should return original message when there no translation

Comment (alexgul4enko) [reply presets/i18n.js:11]: oh this is based on docs. I've just implement all possible variants from original gettext)

Comment (alexgul4enko) [reply presets/i18n.js:11]: https://docs.djangoproject.com/en/3.0/topics/i18n/translation/#using-the-javascript-translation-catalog

Comment (alexgul4enko) [reply presets/i18n.js:11]: https://docs.djangoproject.com/en/3.0/topics/i18n/translation/#gettext-noop

**Pull request merged**